### PR TITLE
NATIVE-482: Add locales for accessibility

### PR DESCRIPTION
**Pull request by NKlug**

### CODE REVIEW

File: locales.json [hunk 1]
```diff
@@ -1690,7 +1690,8 @@
       "noPermission": "Standortzugriff erforderlich",
       "loading": "Bestimme Standort...",
       "notAvailable": "Standortbestimmung deaktiviert",
-      "timeout": "Standortbestimmung fehlgeschlagen"
+      "timeout": "Standortbestimmung fehlgeschlagen",
+      "refresh": "Aktualisiere Orte in der Nähe"
```
Comment (maxammann) [inline locales.json]: Maybe "Liste der Orte erneuern/aktualisieren" would be a better fit.

File: locales.json [hunk 2]
```diff
@@ -1704,7 +1705,8 @@
       "nearbyPlaces": "Nearby locations",
       "noNearbyPlaces": "No nearby locations found.",
       "alert": "Note",
-      "locationError": "To display nearby locations, location services must be active."
+      "locationError": "To display nearby locations, location services must be active.",
```
Comment (maxammann) [inline locales.json:1708]: All keys have to be available at least in German. This is not afaiks

Comment (maxammann) [reply locales.json:1708]: You can also omit the key `locationError` here, as the english text as `notAvailable` is also not yet translated. That way it is maybe more consistent in the end.

TL;DR add this only in error

Comment (steffenkleinle) [reply locales.json:1708]: This is available in german. This is also not used at all.

**Review by steffenkleinle — APPROVED** [review]

File: locales.json (hunk 2 above)
Comment (maxammann) [reply locales.json:1708]: This line is was not changed by @NKlug, my fault, resolving this

File: locales.json (hunk 1 above)
Comment (NKlug) [reply locales.json]: Changed to "Liste der Orte aktualisieren"
